src/data/components/table/DataTable.tsx

```diff
@@ -155,19 +155,16 @@ function ExtendableCell({
   rowStates: RowStates;
   dispatch: Dispatch<Action>;
 }): JSX.Element {
+  const isOpen = () => (rowStates[id] && rowStates[id].height !== 70)
   const classes = useStyles();
-  const [open, setOpen] = useState(
-    rowStates[id] ? rowStates[id].height > 70 : false
-  );
   const openClick = () => {
-    if (open) {
-      dispatch({ type: 'CHANGE_HEIGHT', id: id, height: 70 });
+    if (isOpen()) {
+      dispatch({ type: 'CHANGE_HEIGHT', id, height: 70 });
```

Comment:
her kunne det også bli litt bedre siden det er bare tallet 70 som blir 280 basert på isOpen().
```suggestion
      dispatch({ type: 'CHANGE_HEIGHT', id, height: isOpen() ? 70: 280 });
```